android/app/build.gradle

```diff
@@ -9,47 +9,50 @@ apply plugin: 'com.android.application'
 
 // Set up Android app project.
 android {
-	compileSdkVersion 33
+	ndkVersion project.ext.tiNdkVersion
+	//namespace 'com.titanium.test'
 	defaultConfig {
 		applicationId 'com.titanium.test'
-		minSdkVersion 21
+		compileSdk 33
+		minSdkVersion 26
```

Comment:
Doesn't this bump the default min-sdk version to Android 8? We should remain stable with the current one